tests/binaries/heap_malloc_chunk.c

```diff
@@ -67,12 +80,16 @@ int main(void)
     small_chunk = mem2chunk(before_remainder + 0x210);
     large_chunk = mem2chunk(large);
     unsorted_chunk = mem2chunk(unsorted);
+}
+
+void* thread_func(void* args)
+{
+    // Initialize a 2nd arena
+    malloc(0x18);
```

Comment:
Is the specific size important here? Maybe add a comment about it 😅

Reply:
Nope, just any old chunk to initialize a 2nd arena.